### DISCUSSION TITLE
More convenient visualizations to future releases of DIA-NN

### DISCUSSION THREAD

**Discussion by diabeginner01** [vdemichev/DiaNN#486] [Ideas]
It would be really awesome to get DIA-NN to visualize the results more conveniently than through the --vis command. There's a lot of proteomics users who want to see the integrations of the peaks, thus, a GUI powered visualization module would really help to popularize the software in the community.

**Comment by vdemichev** [comment]
Yes, it's something highly requested!

**Reply by imanhaddad** [reply]
Hello everyone,
I work on a mass spectrometry platform dedicated to proteomics and my colleagues actually wanted to have tools to process or 'see' the results obtained by DIANN. I wrote a little Shiny-like script in the language R , which I share here: https://github.com/imanhaddad/PDD-shiny
There are still few features but all your suggestions are welcome

**Reply by vdemichev** [reply]
That's super cool, thank you for sharing!
